README.md

```diff
@@ -753,6 +755,7 @@ This started as a fork from [node-gitlab-legacy](https://github.com/node-gitlab/
 - [Claudio Vellage](https://github.com/jdalrymple/LRH539)
 - [Seb0uil](https://github.com/seb0uil)
 - [Dylan Taylor](https://github.com/dylanmtaylor)
+- [Niklas Lochschmidt](https://github.com/nlochschmidt)
```

Comment:
Not needed, the all-contributors plugin will add you automatically. I need to go back and convert these old ones

Reply:
Ok, done